frontend/src/js/components/IngestionEvents/IngestionEvents.tsx

```diff
@@ -172,10 +199,22 @@ const renderExpandedRow = (blobStatus: BlobStatus) => {
                 </p></>
             })}
         </div>
+        <h4>All ingestion events prior to extraction</h4>
```

Comment:
Might be worth putting a similar title above the first section e.g. "Extraction events"?

Reply:
good shout, I've also reordered it so that the different sections are in order rather than having extractors first